vsdkx/addon/uncertainty/settings.py

```diff
@@ -0,0 +1,4 @@
+DEFAULT = {
+    "entropy_threshold": 0.67,
```

Comment:
The `entropy_threshold` normally should be on a different config file and not exposed to the outside world. How easy is it to change it?

Reply:
why do we want to change that? we can hard code it in the addon class itself, I don't see any better ideas for meeting that goal. what is reason to hide it from outside environment?

Reply:
I am not a big fun of hardcoding things into the code, it's really bad practice. How about we create two dictionaries inside the `settings.yaml` file where one can be always overwritten by visionx and the other one will contain the defaults of the add on such as `entropy_threhsold`. Also, I believe that this file needs to be renamed to `settings.yaml` instead of `settings.py`

Reply:
I think we discussed and resolved this

Reply:
I think the first part of the comment is still not addressed? If we also addressed it, can you remind me what we agreed on?

Reply:
latest thing we did with respect to putting defaults was to have some parameter value in code for `.get()` method to default to, if the value was not specified through configuration.

about not exposing some values outside, we don't have a mechanism for that which would utilize a yaml file and it doesn't make much sense to have that, if we want to keep it inside addon repository, since it's meant to be installed as a package and any change to config would preactically make a new version, 

so if we want to not expose it to outside, we might as well put it in the code as class attributes, not configurable through configs received from outside, like we already have some values in other repositories.

I agreed with you that hard coding things is considered as a bad practise but I think that in this case, since we don't want to expose things to outside world, not doing that would result in unjustified increase in complexity.

Also, since it's up to us to define what config we should receive as input, we can make it a practice to not send this value as parameter by default and to not include it in configs, by default, but still leave it as a possibility that if it goes wrong for some customer and we still want to change it, then we can put it on that customers specific config to overwrite existting defaults. I think that approach would best suit our existing architecture.

Reply:
okay that sounds logical. However, we need to start documenting these things and make sure that we don't allow visionx to use the `sensitivity_ration` in any of their config files considering that this allows room for error.